exporter/datadogexporter/internal/logs/translator.go

```diff
@@ -58,9 +60,12 @@ const (
 	logLevelFatal = "fatal"
 )
 
+// This constant specifies a tag to be added to all logs sent from the Datadog exporter
+const otelTag = "otel:true"
```

Comment:
nit: I would make this `source:otel` instead. I don't think we would ever have `otel:false` (rather, the absence of `otel:true` is what marks something as not coming from OpenTelemetry), but we could add other `source`s.